source/administration/production-notes.txt

```diff
@@ -740,20 +777,36 @@ accommodate Windows' commitment rules during peak database use.
 MongoDB on Virtual Environments
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This section describes considerations when running MongoDB in some of the
-more common virtual environments.
+This section describes considerations when running MongoDB in some of 
+the more common virtual environments.
 
 For all platforms, consider :ref:`virtualized-disks-scheduling`.
 
-EC2
-```
+:abbr:`AWS (Amazon Web Services)` :abbr:`EC2 (Elastic Compute Cloud)`
+`````````````````````````````````````````````````````````````````````
+
+- When available, enable :abbr:`AWS (Amazon Web Services)`
+  `Enhanced Networking <http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/enhanced-networking.html#enabling_enhanced_networking>`_ 
+  for your instance. Not all instance types support Enhanced Networking. 
+
+  To learn more about Enhanced Networking, see to the 
+  `AWS documentation <http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/enhanced-networking.html#enabling_enhanced_networking>`_.
+
+- Use provisioned IOPS for the storage, with separate devices for
+  journal and data. Do not use the ephemeral storage available on some
+  instance types as their performance changes moment to moment.
+
+- Disable hyperthreading, 
```

Comment:
The suggestions coming from the perf team here are specifically aimed at recommendations for getting reproducible performance tests. They are not aimed at getting maximum performance. As such, they aren't for every body. If a user cares about consistency of performance over raw performance (for benchmarking, or just predictability), they should do this. If they care about performance more than performance reproducibility, they should not pin to one socket and should not disable hyperthreading. 

To some extents that applies to the ephemeral storage too. I don't know if ephemeral storage is better or worse for someone just concerned about performance, but if someone is wants reproducibility they should use the provisioned IOPS.

Reply:
I clarified this.